Examples/Scripts/Python/vertex_fitting.py

```diff
@@ -32,6 +32,109 @@ class VertexFinder(enum.Enum):
     Iterative = (3,)
 
 
+def addVertexFitting(
+    s,
+    field,
+    outputDirRoot: Optional[Union[Path, str]] = None,
+    associatedParticles: str = "associatedTruthParticles",
+    vertexFinder: VertexFinder = VertexFinder.Truth,
+    logLevel: Optional[acts.logging.Level] = None,
+):
+    """This function steers the vertex fitting
+
+    Parameters
+    ----------
+    s: Sequencer
+        the sequencer module to which we add the Seeding steps (returned from addVertexFitting)
+    field : magnetic field
+    outputDirRoot : Path|str, path, None
+        the output folder for the Root output, None triggers no output
+    logLevel : acts.logging.Level, None
+        logging level to override setting given in `s`
+    """
```

Comment:
- [ ] Missing `associatedParticles` in docstring
- [ ] Missing `vertexFinder` in docstring

Reply:
Added. Thanks for reporting.